app/static/src/app/userMessages.ts

```diff
@@ -36,7 +36,16 @@ export default {
         selectParamToVary: "Please select at least one parameter to vary during model fit.",
         compileRequired: "Model code has been updated. Compile code and Fit Model for updated best fit.",
         fitRequired: "Model code has been recompiled, or options or data have been updated. "
-            + "Fit Model for updated best fit."
+            + "Fit Model for updated best fit.",
+        fitRequirements: {
+            prefix: "Cannot fit model. Please",
+            unknown: "contact the administrator, as this is unexpected",
+            needsModel: "compile a model (code tab)",
```

Comment:
Should maybe capitalise all the tab names to match the actual labels on the tabs